src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,12 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
In the readme file first point under the task states that-
Display a calendar in the middle of the screen. Use SCSS and follow BEM. Don't use JS.
Please check what can you do.

Reply:
Centered the calendar.